quinn/src/connection.rs

```diff
@@ -79,29 +79,47 @@ impl Connecting {
 
     /// Convert into a 0-RTT or 0.5-RTT connection at the cost of weakened security
     ///
-    /// Opens up the connection for use before the handshake finishes, allowing the API user to
-    /// send data with 0-RTT encryption if the necessary key material is available. This is useful
-    /// for reducing start-up latency by beginning transmission of application data without waiting
-    /// for the handshake's cryptographic security guarantees to be established.
+    /// Returns `Ok` immediately if the local endpoint is able to attempt 0/0.5-RTT conversion. If
```

Comment:
"conversion" is strange word choice here. Maybe "communication" or "... able to attempt sending 0/0.5-RTT data"?